### ISSUE TITLE
The graalVM fails to build a native-image

### ISSUE THREAD

**Issue by SybrStoker** [SybrStoker/password-generator-cli#7]
By today i've been strugling for 5 days with this problem. Now im gonna try to cut some methods to understand what part of code causing trouble. At first i had trouble to build a fatJar to compile a native-image with all the libraries. Now on this version it doesn't want to compile the image because of java.util.awt. When i tryed to build this project with micronaut then it told me that compiler is stuck in  an infinit loop or something. Im not gonna provide all the error log here. I write it just for myself anyway.

**Comment by SybrStoker** [comment]
A little update here. It turns out really intersting. My program is stuck in a deadlock during compilation for some reason. I found out yesterday nigth that this wierd loop is a deadlock. I didn't pay attention to the log graalVM provided me with. Ah, today i have read more about what it is and now im gonna take action on finding out where the rat is hiding. 

- [x] Im gonna try to compile a hello-world binary from jar file to make sure the graalVM works correctly.

- [x] Im gonna try to compile a hello-world with picocli to make sure there is no conflict.

if all that will compile successfully then the problem is in Generator.java. I have already got rid of Features to see if the problem in awt library but it's not. This is my first time dealing with a deadlock.  I just found out what it is today lol. Oh, i might take another week or so.

**Comment by SybrStoker** [comment]
I did first two steps and what do u know? The hello world doesn't compile with a picocli. I used a micronaut project for this where i chose a graalVM support. Idk what to do now. I trully belived tha my code was breaking the compilation. I suppose the new version of graalVM breakes the picocli framework. Maybe my laptop is too weak for this and this is the problem but it would not break on second step then(Analyzing). I will try to use an older version of graalVM and check if it  helps. if it won't i can't solve this problem on my own just yet. I will simply move on and leave this issue here for good. Man this sucks. Maybe i should use another cli framework while it's not too late. I also will try to compile this code on my main machine in the next month(rn im stuck with my old laptop).
